packages/core/theme/src/components/alert.ts

```diff
@@ -0,0 +1,92 @@
+import type {VariantProps} from "tailwind-variants";
+
+import {tv} from "../utils/tv";
+/**
+ * Alert wrapper **Tailwind Variants** component
+ *
+ * @example
+ * ```js
+ * const {base, mainWrapper, title, description} = input({...})
+ *
+ * <div className={base()}>
+ *    <AlertIcon/>
+ *    <div className={mainWrapper()}>
+ *       <div className={title()}>Title</div>
+ *       <div className={description()}>Description</div>
+ *   </div>
+ *    <button>
+ *      // close button
+ *    </button>
+ * </div>
+ * ```
+ */
+
+const alert = tv({
+  slots: {
+    base: ["flex flex-row w-[342px] h-17  p-3"],
+    title: ["text-medium font-normal block h-6"],
+    description: ["text-small font-normal h-5"],
+    mainWrapper: ["w-[268px] h-11 ms-5 flex flex-col box-border items-start"],
```

Comment:
try not to px as unit

Reply:
should I replace as follows:
w-[342px] -> w-114
w-[268px[ -> w-89